hystrix/settings.go

```diff
@@ -80,12 +86,18 @@ func ConfigureCommand(name string, config CommandConfig) {
 		errorPercent = config.ErrorPercentThreshold
 	}
 
+	queueSizeRejectionThreshold := DefaultQueueSizeRejectionThreshold
```

Comment:
your PR comment mentions that the default is the same as `MaxConcurrentRequests`, but in fact it is statically set to `50` here even if the user provides a different concurrency setting.

i like the idea of having the queue size (if unset) be equal to the concurrency setting, which would change the code here to remove `DefaultQueueSizeRejectionThreshold` and replace it with `max`

Reply:
Fixed.